src/locales/de/PAGES/EXPLORE.json

```diff
@@ -0,0 +1,18 @@
+{
+    "TITLE": "Schritt 2: Die zu prüfende Website erkunden",
+    "INTRO": "Erkunden Sie die Website, um ihren Zweck, ihre Funktionen und Benutzung zu verstehen. Dieser Schritt hilft Ihnen, Seiten auszuwählen für die Prüfung in den folgenden Schritten. Identifizieren Sie die Techniken, die auf der Website benutzt werden, und notieren Sie andere Aspekte, falls gewünscht. Normalerweise ist es am Besten, den Besitzer und die Entwickler der Website in diesem Schritt einzubeziehen.",
+    "RESOURCE_LINK_NAME": "Step 2: Explore the Target Website",
+    "HD_RELIEDUP_TECH": "Benutzte Web-Techniken",
+    "INF_RELIEDUP_TECH": "Identifizieren Sie die Web-Techniken, auf die [man sich verlässt im Sinne der WCAG 2](https://www.w3.org/Translations/WCAG20-de/#reliedupondef) um die Website zur Verfügung zu stellen. Für weitere Informationen, siehe <a hreflang='en' lang='en' href='https://www.w3.org/TR/WCAG-EM/#step2d'>WCAG-EM Step 2.d: Identify Web Technologies Relied Upon</a>.",
+    "LABEL_TECH": "Web-Technik",
+    "LABEL_TECH_SPEC": "Adresse der Spezifikation (URL)",
+    "PLH_TECH_SPEC": "Web-Adresse (URL) oder Beschreibung der Spezifikation der Web-Technik",
+    "BTN_ADD_TECH": "Web-Technik hinzufügen",
+    "HD_NOTE_TAKING": "Optionale Notizen zur Erkundung",
+    "INF_NOTE_TAKING": "Dieser Abschnitt erlaubt Ihnen, Notizen während Ihrer Erkundung zu machen. Diese Notizen werden im nächsten Schritt angezeigt, um Ihnen zu helfen eine Auswahl von Seiten für die Prüfung zu treffen. **Diese Notizen sind nicht im finalen Bericht enthalten**. Sie werden gespeichert wenn Sie die Datei speichern und später mit der Datei wieder geladen.",
+    "LABEL_ESSENT_FUNC": "Wesentliche Funktion der Website",
+    "INF_ESSENT_FUNC": "Sie können dieses Feld benutzen, um Notizen zur wesentlichen Funktion der Website zu machen. Beispiele von wesentlichen Funktionen enthalten:<ul><li>„Ein Produkt im Shop-Bereich der Website auswählen und kaufen“</li><li>„Ein Formular auf der Website ausfüllen und abschicken”</li><li>„Ein Konto auf der Website registrieren”</li></ul> Für weitere Informationen, siehe <a hreflang='en' lang='en' href='https://www.w3.org/TR/WCAG-EM/#step2b'>WCAG-EM Step 2.b: Identify Essential Functionality of the Website</a>.",
+    "LABEL_VARIETY_PAGE_TYPES": "Vielfalt der Seitenarten",
+    "INF_VARIETY_PAGE_TYPES": "Sie können dieses Feld benutzen, um Notizen zu den Arten (im Gegensatz zu Instanzen) von Seiten machen, die Sie auf der Website finden. Das beinhaltet Notizen zu verschiedenen Stilen, Layouts, Strukturen und Funktionen, die auf der Website verwendet werden. Für weitere Informationen, siehe <a hreflang='en' lang='en' href='https://www.w3.org/TR/WCAG-EM/#step2c'>WCAG-EM Step 2.c: Identify the Variety of Web Page Types</a>.<br> **Hinweis**: „Seiten“ beinhalten verschiedene <a hreflang='en' lang='en' href='https://www.w3.org/TR/WCAG-EM/#states'>Zustände von Webseiten (Englisch)</a>.",
```

Comment:
I don't really understand this sentence (as a native German speaker):

"Sie können dieses Feld benutzen, um Notizen zu den Arten (im Gegensatz zu Instanzen) von Seiten machen, die Sie auf der Website finden."

That's probably a 1:1 translation of the original, but I can't make sense out of it.

Reply:
Guter Punkt, das klingt wirklich seltsam und schwierig nachzuvollziehen.

Hättest Du einen Vorschlag?

Vielleicht Seitenvorlagen oder Templates?

Reply:
Vielleicht "Sie können dieses Feld benutzen, um Notizen zu einzelnen Seiten zu machen", falls das sinngemäss korrekt ist. Ich glaube einfach, der Satz ist ein wenig zu kompliziert, deswegen habe ich auch schon mühe zu verstehen, was gemeint ist.

Reply:
Hm. Genau das ist es ja leider eben nicht, einzelne Seiten.

Das Label habe ich mit „Vielfalt der Seitenarten” übersetzt. Das klingt ja auch schon komisch, oder?

Vorschlag: „Überblick über Seitentypen”?

https://www.w3.org/TR/WCAG-EM/#step2c

Es geht ja darum dass einzelne Seiten oft so ähnlich sind, dass es nicht sinnvoll ist, jede einzelne zu prüfen. Es reicht dann aus, ein „Exemplar“ eines Seitentypen zu prüfen. Vielleicht ist das besser?

Seiten sind vom gleichen Typ, wenn sie sich quasi nur in Text- und Bildinhalt unterscheiden.

Statt Vielfalt wäre vielleicht Spektrum besser? „Spektrum von Seitentypen”. Oder ist das noch zu abstrakt?